collectors/ClusterPropertiesCollector.py

```diff
@@ -23,6 +23,14 @@ def collect(self):
             logger.warning(f'skipping {self.target} in {self.name}, no token')
             return
 
+        http_response_code, http_gauge = self.create_http_response_metric(self.target, token, self.name)
+        logger.debug(f'HTTP response code in {self.name} for {self.target}: {http_response_code}')
+        yield http_gauge
+
+        if http_response_code > 200:
+            logger.critical(f'HTTP response code in {self.name} for {self.target}: {http_response_code}, no return')
```

Comment:
I would move this log line and check for 200 to `create_http_response_metric()` to avoid duplicates. Same for `logger.debug`
Just return the gauge to yield it and a `True` or `False` variable which we use for continuing, like:

```
api_responding, http_gauge = self.create_http_response_metric(..)
if not api_responding:
    return
```